CHANGELOG.md

```diff
@@ -0,0 +1,334 @@
+  * Uncertainty propagated according to uncertainties in wind speed and in the ratio of IME divided by plume length. IME/L is treated as a signal variable.
+### GAO & ANG 
+#### CH4
+##### L2b (collection: l2b-ch4-mf-v2)
+* Wave length window
+  * (2100, 2480)
+* Bad pixel saturation thresholds
+  * 2092-2098nm range: Threshold of 4
+  * **<span style="color:red"> 2400-2500nm range: Threshold of 1 </span>**
+* Misc Changes
+  * **<span style="color:red"> Rotation has been removed and rasters are now oriented northward </span>**
+##### L3a (collection: l3a-ime-ch2-mf-v2)
+* Uses static ppm-m threshold
+  * 1000ppm-m
+* Concentric Circles IME Method
+  * Max Fetch 284m
+  * **<span style="color:red"> Include center pixel for IME and plume length quantification </span>**
+##### L4a
+* Wind products used
+  * HRRR data archive for plumes within the HRRR coverage area 
+  * ECMWF_IFS or ERA5 from OpenMeteo globally
+  * **<span style="color:red"> HRRR winds are spatially averaged to match the resolution of ECMWF_IFS allowing for cross comparisons of plume uncertainties regardless of detection location </span>**
+* Emission Quantification Uncertainty
+  * Uncertainty propagated according to uncertainties in wind speed and in the ratio of IME divided by plume length. IME/L is treated as a signal variable.
+#### CO2
+##### L2b (collection: l2b-co2-mfal-v2)
+**<span style="color:red"> This is a change from previously used mfa to mfal products </span>**
+
+* Wave length windows
+  * (1860, 2190)
+* Bad pixel saturation thresholds
+  * **<span style="color:red"> 2092-2098nm range: Threshold of 4 </span>**
+  * **<span style="color:red"> 2400-2500nm range: Threshold of 1 </span>**
+* Unit Absorption Spectrum 
+  * Solar Zenith Angle is queried
+  * **<span style="color:red"> Column Water Vapor is now queried </span>**
+  * **<span style="color:red"> Elevation is now queried </span>**
+* Misc Changes
+  * **<span style="color:red"> Rotation has been removed and rasters are now oriented northward </span>**
+
+##### L3a (collection: l3a-co2-mfal-v2)
+* Uses dynamic threshold masking 
+  * **<span style="color:red"> Changed dynamic mask fit line </span>**
+* Concentric Circles IME Method
+  * Maximum Fetch is 284m
+  * **<span style="color:red"> Include center pixel for IME and plume length quantification </span>**
+##### L4a
+* Wind products used
+  * HRRR data archive for plumes within the HRRR coverage area 
+  * ECMWF_IFS or ERA5 from OpenMeteo globally
+  * **<span style="color:red"> HRRR winds are spatially averaged to match the resolution of ECMWF_IFS allowing for cross comparisons of plume uncertainties regardless of detection location</span>**
+* Emission Quantification Uncertainty
+  * Uncertainty propagated according to uncertainties in wind speed and in the ratio of IME divided by plume length. IME/L is treated as a signal variable.
+
+## Version 2 (Visualization Products Only) <a name="version2"></a>
+### l2b-rgb-v2 
```

Comment:
Nit: it's a little jarring that these collection names are in a different font size than the ones above